README.md

```diff
@@ -77,3 +78,6 @@ Install it with:
 `sudo gem install gherkin_lint`
 
 After that `gherkin_lint` executable is available.
+
+## Configuration
+If you have a custom configuration you'd like to run on a regular basis instead of passing enable and disable flags through the CLI on every run, you can configure a ```.gherkin_lint.yml``` file that will be loaded on execution.  The format and available linters are in [```config/default.yml```](config/default.yml)
```

Comment:
would like if there is a `gherkin_lint --auto-gen-config` similar to rubocop. Nobody knows where gherkin_lint is installed and where the config/default.yml is located. It is also difficult to Keep the config/default.yml up to date, when addind a new Feature. An generated config may handle this better.